### PR TITLE
feat(artifactory,artifactory-ha): Add logger to nginx

### PR DESCRIPTION
**Pull request by k**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Adds same logging side cars in #206 to nginx as well

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -476,6 +476,9 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.image.repository`    | Container image                   | `docker.bintray.io/jfrog/nginx-artifactory-pro`        |
 | `nginx.image.version`       | Container version                 | `.Chart.AppVersion`                                    |
 | `nginx.image.pullPolicy`    | Container pull policy             | `IfNotPresent`                                         |
+| `nginx.logger.image.repository`| repository for logger image | `busybox`                                           |
+| `nginx.logger.image.tag`       | tag for logger image | `1.30`                                                     |
```
Comment (eldada) [inline stable/artifactory-ha/README.md]: How about merging this with the `artifactory.logger.image.repository` and `artifactory.logger.image.repository` and giving a common name like `logger.image.repository` and `logger.image.tag` or just using the same as artifactory?

Comment (k) [reply stable/artifactory-ha/README.md]: What do we do about the different logger names though between artifactory and nginx? Should we have the top level logger only be for the image and then the logger in nginx and artifactory specify the names of the logs?

Comment (eldada) [reply stable/artifactory-ha/README.md]: Yes. They should be as you set them until now. Just the image properties should be common.

**Comment by eldada** [comment]
@k - can you rebase and complete this?

**Comment by k** [comment]
Alright @eldada should be good to go

File: stable/artifactory-ha/README.md [hunk 2]
```diff
@@ -348,7 +348,7 @@ helm install --name artifactory-ha --set imagePullSecrets=regsecret jfrog/artifa
 
 ### Logger sidecars
 
-This chart provides thee option to add sidecars to tail various types of logs from artifactory. To see the potential values check the `artifactory.logger.names` value in `values.yaml`
+This chart provides thee option to add sidecars to tail various types of logs from artifactory. To see the potential values check the `artifactory.loggers` and `nginx.loggers` values in `values.yaml`
```
Comment (eldada) [inline stable/artifactory-ha/README.md]: Fix: Change `thee` to `the`.

**Comment by k** [comment]
rebased and typo fixed

**Pull request merged**